### DISCUSSION TITLE
G5 raw sensor value of 0

### DISCUSSION THREAD

**Discussion by Kirley001** [NightscoutFoundation/xDrip#2249] [Q&A]
![inbound2632720338316732172](https://user-images.githubusercontent.com/110271822/181870447-a530e607-cb33-436a-adf1-67dbfb322d60.jpg)
![inbound8744112359005039377](https://user-images.githubusercontent.com/110271822/181870460-c1a2569e-a980-4c22-93a9-23101a1faec2.jpg)
Recently I started to receive "raw sensor value of 0" alerts on a previously working sensor. I thought it may be a bluetooh issue so I attempted to troubleshoot but after no avail, I decided to replace the sensor and to much disappointment, I continued to recieive the same alerts. I actually did this twice and can't seem to get it working. Any insight is greatly appreciated!

**Comment by Navid200** [comment]
Thanks for reporting

@jamorham What could cause the raw sensor value of 0 on multiple new sensors?

**Comment by Navid200** [comment]
Have the sensor passed their expiry date?

**Reply by Kirley001** [reply]
They were not, but, hard resetting the transmitter seemed to work.
